apps/site/assets/ts/stop/__tests__/DepartureListTest.tsx

```diff
@@ -268,4 +268,52 @@ describe("DepartureList", () => {
     expect(screen.queryByText("Cancelled")).not.toBeInTheDocument();
     expect(screen.getByText("25 min")).toBeInTheDocument();
   });
+
+  it("should hide upcoming alerts on other parts of the route", () => {
+    const alerts = [
```

Comment:
nitpick: can a delay alert be added to test that it is not rendered on the page?